src/lib/holocene/api-pagination.svelte

```diff
@@ -29,11 +31,14 @@
     itemsKeyname?: string;
     previousButtonLabel: string;
     nextButtonLabel: string;
+    filterable?: boolean | undefined;
+    filterInputLabel?: string | undefined;
```

Comment:
Since the input only shows up if `filterable && filterInputLabel`, can we make `filterInputLabel` required if `filterable` is `true`?